recipe/meta.yaml

```diff
@@ -19,9 +21,16 @@ requirements:
   build:
     - make  # [unix]
     - cmake
+    - ninja  # [win]
     - pkg-config  # [not win]
     - {{ compiler('cxx') }}
     - file  # [arm64]
+    - zip  # [not win]
+    - unzip  # [not win]
+    - curl  # [not win]
+    - tar  # [not win]
```

Comment:
Just to confirm: are these new build dependencies required by vcpkg?

Reply:
[Yes.](https://github.com/microsoft/vcpkg#installing-linux-developer-tools)